client/src/featureform/get.py

```diff
@@ -1,252 +1,211 @@
-            print("{:<30} {:<35}".format("NAME", "VARIANT"))
+            format_rows([("NAME: ", x.name),
+            ("DESCRIPTION: ", x.description),
+            ("TYPE: ", x.type),
+            ("SOFTWARE: ", x.software),
+            ("TEAM: ", x.team),
+            ("STATUS: ", x.status.Status._enum_type.values[x.status.status].name)])
+            format_new_para("SOURCES:")
+            format_rows("NAME", "VARIANT")
             for s in x.sources:
-                print("{:<30} {:<35}".format(s.name, s.variant))
-            print("")
-            print("FEATURES:")
-            print("{:<30} {:<35}".format("NAME", "VARIANT"))
+                format_rows(s.name, s.variant)
+            format_new_para("FEATURES:")
+            format_rows("NAME", "VARIANT")
             for f in x.features:
-                print("{:<30} {:<35}".format(f.name, f.variant))
-            print("")
-            print("LABELS:")
-            print("{:<30} {:<35}".format("NAME", "VARIANT"))
+                format_rows(f.name, f.variant)
+            format_new_para("LABELS:")
+            format_rows("NAME", "VARIANT")
             for l in x.labels:
-                print("{:<30} {:<35}".format(l.name, l.variant))
-            print("")
-            print("TRAINING SETS:")
-            print("{:<30} {:<35}".format("NAME", "VARIANT"))
+                format_rows(l.name, l.variant)
+            format_new_para("TRAINING SETS:")
+            format_rows("NAME", "VARIANT")
             for t in x.trainingsets:
-                print("{:<30} {:<35}".format(t.name, t.variant))
-    except grpc._channel._MultiThreadedRendezvous:
-        print("Provider not found.")
-
-def GetEntity(stub, name):
-    searchName = metadata_pb2.Name(name=name)
-    try:
-        for x in stub.GetEntities(iter([searchName])):
-            print("{:<20} {:<15}".format("NAME: ", x.name))
-            print("{:<20} {:<15}".format("STATUS: ", x.status.Status._enum_type.values[x.status.status].name))
-            print("VARIANTS:")
-            print("{:<20} {:<25}".format(x.default_variant, 'default'))
-            for v in x.variants:
-                if v != x.default_variant:
-                    print("{:<20} {:<35}".format(v, ''))
-    except grpc._channel._MultiThreadedRendezvous:
-        print("Entity not found.")
-
-def GetModel(stub, name):
-    searchName = metadata_pb2.Name(name=name)
-    try:
-        for x in stub.GetModels(iter([searchName])):
-            print("{:<20} {:<15}".format("NAME: ", x.name))
-            print("{:<20} {:<15}".format("STATUS: ", x.status.Status._enum_type.values[x.status.status].name))
-            print("VARIANTS:")
-            print("{:<20} {:<25}".format(x.default_variant, 'default'))
-            for v in x.variants:
-                if v != x.default_variant:
-                    print("{:<20} {:<35}".format(v, ''))
+                format_rows(t.name, t.variant)
+            format_new_para("")
+            return x
```

Comment:
Is there a reason why this returns after one iteration?

Reply:
it just returns the first in the stream